Classes/S3Storage.php

```diff
@@ -68,7 +68,7 @@ class S3Storage implements WritableStorageInterface
     protected $s3Client;
 
     /**
-     * @Flow\InjectConfiguration("profiles.default")
+     * @Flow\InjectConfiguration(package="Flownative.Aws.S3", path="profiles.default")
```

Comment:
Not really related to this change, but I need to extend this class and without the package configuration the extended class try to inject property from the wrong package, I think it's a good practice to include package and path with injecting configuration.

Reply:
Hm… what I did (once) in the past was add that property with the inject annotation (including package) to my custom class.

I think calling this good practice is not really true… I'd rather like it, when the processing of such an annotation looked at the actual package the annotated file is in, when determining the default. Then it would work as expected, but you could still override it in your custom class (which makes more sense, semantically, IMHO).

What do you think?